protocol/testing/containertest/preupgrade_entrypoint.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+set -eo pipefail
+
+# This entrypoint configures the chain to start in a preupgrade state before forwarding args to cosmovisor.
+# The following need to be done:
+#   - Set the preupgrade binary (downloaded in containertest.sh) to be the genesis binary
+#   - Set the binary at the current commit to  be the ugprade binary
+#   - Set the genesis to be the preupgrade genesis
+
+if [[ -z "${UPGRADE_TO_VERSION}" ]]; then
+    echo >&2 "UPGRADE_TO_VERSION must be set"
+    exit 1
+fi
+
+MONIKERS=(
```

Comment:
I could pass in moniker as an env variable as well. Choosing not to do so for now.